### PR TITLE
Book list

### PR DESCRIPTION
**Pull request by Takhmina175**
We use DOM manipulation to create HTML tags.
- Create functions like addBook and deletBook to add and delete books
- Use LocalStorage to convert objects into strings.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  createBook(newBook);
+  titleInput.value = '';
+  authorInput.value = '';
+};
+
+function deleteBook(el) {
+  if (el.classList.contains('delete')) {
+    el.parentElement.remove();
+  }
+}
```
Comment (Dipeshtwis) [inline index.js]: - [ ] After removing the element and refreshing the browser the element still persists. Please make sure to update the localstorage after removing the element.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Takhmina175 and @Qoosim, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: You have implemented add function perfectly.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+const authorInput = document.querySelector('#author');
+const bookContainer = document.querySelector('#ptext');
+const addBtn = document.getElementById('add-btn');
+
+let books = [];
+if (localStorage.getItem('books') === null) {
+  books = [];
+} else {
+  books = JSON.parse(localStorage.getItem('books'));
+}
```
Comment (Lameck1) [inline index.js:11]: - [OPTIONAL] We can refactor this to one line of code like this 👇🏼 using the OR(||) operator.

```suggestion
let books = JSON.parse(localStorage.getItem('books')) || [];
```

**Review by Lameck1 — APPROVED** [review]
Hi, @Takhmina175 & @Qoosim 

You've done well implementing the requested change 💯!  Your project is now complete and it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**